src/components/Site/Site.jsx

```diff
@@ -74,6 +74,7 @@ class Site extends React.Component {
           toggle={this._toggleSidebar} /> : null}
 
         <Switch>
+          <Route exact strict path="/:url*" render={props => <Redirect to={`${props.location.pathname}/`}/>} />
```

Comment:
Why is this needed?

Reply:
The context is in the issue, we have two types of Urls, and ssr urls are different from react router urls. When users copy links and share them they share react router links without trailing slashes, which gets into redirect and sometimes anchors didnt work after that e.g. `/concepts#something` ended up on `/concepts/#something`. Also reporting in GA is broken, all page views are duplicated with `/concepts` (landed via react-router) and `/concepts/` (landed from search)